x-pack/plugins/alerting/common/routes/backfill/apis/schedule/schemas/v1.ts

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { schema } from '@kbn/config-schema';
+import { backfillResponseSchemaV1, errorResponseSchemaV1 } from '../../../response';
+
+export const scheduleBodySchema = schema.arrayOf(
+  schema.object(
+    {
+      rule_id: schema.string(),
+      start: schema.string(),
+      end: schema.maybe(schema.string()),
+    },
+    {
+      validate({ start, end }) {
+        const parsedStart = Date.parse(start);
+        if (isNaN(parsedStart)) {
+          return `Backfill start must be valid date`;
+        }
+
+        if (end) {
+          const parsedEnd = Date.parse(end);
+          if (isNaN(parsedEnd)) {
+            return `Backfill end must be valid date`;
+          }
+          const startMs = new Date(start).valueOf();
+          const endMs = new Date(end).valueOf();
+          if (endMs <= startMs) {
+            return `Backfill end must be greater than backfill start`;
+          }
+        }
+      },
+    }
+  ),
+  { minSize: 1, maxSize: 10 }
```

Comment:
Open to feedback re this initial limit!

Reply:
So the request limit is 10?  My initial read on the "limit" was that was the total number of backfill requests the system would honor, but that doesn't make a lot of sense.  Presumably unlimited?  If so, 10 seems kinda small.  You could imagine someone selecting all the security rules (hundreds) and doing "backfill", which would take forever, manually.

Maybe 100 is a little more reasonable?